.github/actions/report-job-status/action.yml

```diff
@@ -2,36 +2,31 @@ name: Send VK Teams notification on failure
 description: Send the message to VK Teams chat on failure
 
 inputs:
-  api-url:
-    required: true
-    description: VK Teams API URL
   bot-token:
     required: true
     description: VK Teams bot token
-  chat-id:
```

Comment:
By the way, you could just define the default value here. This action is planned to be used in other `tarantool` related repos and on one day we can decide to use different chats for those repos.

Reply:
Looks like fixed.